File: C_Sharp_code/Example/WindowsFormsApp/VB_demo03/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim classA As New ClassA
        classA.str = "Hello"
        Console.WriteLine(classA.str)
    End Sub

    Public Class ClassA
        Public Property str As String
    End Class

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Dim classB As New ClassB
        classB.x = "this's "
        classB.y = "VB!"
        Console.WriteLine(classB.x + classB.y)
    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo03/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CS_deom03
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            ClassA classA = new ClassA();
            classA.str = "Hello";
            Console.WriteLine(classA.str);
        }
        public class ClassA
        {
            public string str;
        }

        private void button2_Click(object sender, EventArgs e)
        {
            ClassB classB = new ClassB();
            classB.x = "hi,";
            classB.y = "this's partial!";
            Console.WriteLine(classB.x+classB.y);
        }
    }
}
